### DISCUSSION TITLE
为什么限制进展补录一个月30次

### DISCUSSION THREAD

**Discussion by TomIsFat** [WatsonYao/nian_app#44] [Q&A] [answered]
我会频繁的使用nian的进展补录功能，因为想精准的记录我什么时间吃了什么，从而更好的观察什么东西不应该吃（拉肚子后查看）
然而吃完东西可能并不会马上想起来记录，所以会手动选择进展补录 ...

所以很疑惑为什么限制进展补录次数，希望能够不限次数（或者限制放的更宽一点？

**Comment by WatsonYao** [comment, marked as answer]
限制次数就是为了培养随时记录的好习惯，未来会考虑放宽到50次。
